### ISSUE TITLE
Model appears tiny (minuscule) when imported in Cura

### ISSUE THREAD

**Issue by scooper4711** [stockto2/desktophero#89]
# Steps to recreate:
1. Open & log in  to Desktophero.com
2. Press "Download to STL with the default character
3. Open default character in e.g. Cura 2.3.1
4. Notice the size. The height of the figure is 8mm

# Expected results
- character should be approximately 28mm high.

**Comment by stockto2** [comment]
Yep - just need to scale the character up during the export. Around here somewhere: https://github.com/stockto2/desktophero/blob/a5a1218b2fec827b22438a4f16a9c062b229e30c/htdocs/vendor/threejs/external/STLExporter.js#L111

**Comment by espray** [comment]
I would not recommend changing 3rd party libs, it will limit your ablity to update the lib.
Mesh.scale() should do the trick
http://jsfiddle.net/EtSf3/3/
If Cura said the height was 8mm, then a scale up by 3.5 should do the trick to export to 28mm

Future: Some UI for selecting figure height static options could be 6, 15, 18, 25, 28, 30 or just allow user to enter height in mm before export STL export

**Comment by ElsinoreBrewmaster** [comment]
**Scale models to 480%** in your slicer for a standard size D&D or Warhammer scale figure. I measured a retail figure with calipers and then calculated that multiplier in Cura. Then I printed a figure and measured it.  Each figure's proportions (head size, etc.) were different, but the overall dimensions matched well. After a lot of research, I found that there is no real standard for model proportions, even within figures from the same manufacturer, so as long as the figure heights are close, they are scaled as appropriately as retail minis.

**Comment by stockto2** [comment]
Sweet! Thanks for that info.
